src/Application/GamePathResolver.cpp

```diff
@@ -56,17 +61,17 @@ static std::vector<std::string> resolvePaths(Environment *environment, const Pat
     result.push_back(".");
```

Comment:
Apologies, not had a chance to look through this yet. Hopefully soon. One thing I can see from the output is that we might need to exclude this from windows environments. That message should output the working directory in the event of not finding other paths.

Reply:
replace with `std::filesystem::current_path().string()`